custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+name: Litter Box
+products:
+  - id: ebf44398a24b7b6be3vnml
+    name: DOEL Ti+
+primary_entity:
+  entity: sensor
+  name: Cat Weight
+  icon: "mdi:cat"
+  class: weight
+  dps:
+    - id: 6
+      type: integer
+      name: sensor
+      unit: kg
+      class: measurement
+      mapping:
+        - scale: 10
+    - id: 134
+      type: integer
+      name: sensor
+      unit: lb
+      class: measurement
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: sensor
+    name: Visits
+    icon: "mdi:emoticon-poop"
+    dps:
+      - id: 7
+        type: integer
+        name: sensor
+        unit: visits
+  - entity: sensor
+    name: Visits Duration
+    icon: "mdi:paper-roll"
+    class: duration
+    dps:
+      - id: 8
+        type: integer
+        name: sensor
+        unit: seconds
+        class: measurement
+  - entity: binary_sensor
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 22
+        type: bitfield
+        name: sensor
+        mapping:
+          - dps_val: 0
+            value: false
+          - value: true
+      - id: 22
+        type: bitfield
+        name: fault_code
+  - entity: button
+    name: Clean Cycle
```

Comment:
The word Cycle seems redundant. I think just "Clean" and "Empty" would be better names for these buttons.